aarone_frontend/src/app/page_sections/footer/footer1/footer1.component.html

```diff
@@ -41,33 +41,32 @@ <h4 class="text-uppercase m-b10 m-t0">Subscribe to our newsletter!</h4>
                     <div class="widget widget_about">
                         <!--<h4 class="widget-title">About Company</h4>-->
                         <div *ngIf="page != '4'" class="logo-footer clearfix p-b15">
-                            <a routerLink="/index"><img src="assets/images/logo-dark.png" alt=""></a>
+                            <a routerLink="/index"><img src="assets/images/logo-removebg-preview.png" alt=""></a>
+                            <!-- <a routerLink="/index"><img src="assets/images/logo_dark-removebg-preview.png" alt=""></a> -->
```

Comment:
It seems like this comment was added? is that intentional, or did I read the diff wrong?